main/EDPftvarcon.F90

```diff
@@ -49,8 +49,7 @@ module EDPftvarcon
                                                      ! that is occupied by crown. For fire model. 
      real(r8), allocatable :: bark_scaler(:)         ! scaler from dbh to bark thickness. For fire model.
      real(r8), allocatable :: crown_kill(:)          ! crown resistance to fire. (1 = none) For fire model.
-     real(r8), allocatable :: active_crown_fire(:)   ! Is plant susceptible to active crown fire?(1=yes,0=no)
-     real(r8), allocatable :: foliar_moisture(:)     ! foliar moisture content from dry fuel [%]
```

Comment:
Strange, it wants to remove foliar_moisture, but foliar_moisture is not in master.. I wonder if git is confused...?